libpromises/cf3.defs.h

```diff
@@ -132,12 +132,6 @@ typedef enum
 
 /*****************************************************************************/
 
-#define CF_FAILEDSTR "BAD: Unspecified server refusal (see verbose server output)"
-#define CF_CHANGEDSTR1 "BAD: File changed "     /* Split this so it cannot be recognized */
-#define CF_CHANGEDSTR2 "while copying"
```

Comment:
We figured it out, it's likely to avoid the string being recognised when transferring a CFEngine binary. It's terrible and hacky, but the way the protocol is designed it can't really be helped.

Reply:
...with other words I need to revert this. It fits better where I moved it anyway, so won't be a big change.